### ISSUE TITLE
Interpolation setting "closest" not imported from glTF (Collada works)

### ISSUE THREAD

**Issue by fhd** [godotengine/godot#75209]
### Godot version

v3.5.1.stable.mono.official [6fed1ffa3]

### System information

Ubuntu 22.04.2 LTS, GLES3, GeForce RTX 3090

### Issue description

I've created a plane mesh with a texture applied to it in Blender, setting interpolation from the default (_linear_) to _closest_, to have the texture render without any undesired edge pixels.

When I export that mesh as glTF, Godot renders it as if interpolation was set to _linear_ instead of _closest_. Exporting the exact same mesh as Collada, Godot renders it as desired. I've also looked at the exported glTF in a different viewer application and it renders as expected there, so I suspect an issue with Godot's glTF importer.

Here's the original texture:
![test](https://user-images.githubusercontent.com/383120/226833451-0841c2fe-e8e2-466c-91e6-f590342d695a.png)

Here's how it looks in Blender with interpolation set to _linear_:
![Screenshot from 2023-03-22 08-38-11](https://user-images.githubusercontent.com/383120/226833160-56140bbe-9638-43d3-ba23-266f93c5b6c6.png)

Here's how it looks in Blender with interpolation set to _closest_:
![Screenshot from 2023-03-22 08-38-21](https://user-images.githubusercontent.com/383120/226833177-befe2cbd-5be1-4379-ae29-628f617d7046.png)

Here's how it looks in Godot - on the left is the glTF (renders incorrectly), on the right the Collada (renders correctly):
![Screenshot from 2023-03-22 08-39-35](https://user-images.githubusercontent.com/383120/226833191-79ab4a10-4bf3-4423-8d76-38825ac4d126.png)

### Steps to reproduce

# In Blender

1. Create a new plane mesh
2. In the material pane, create a new material
3. Set the new material's base colour to an image texture
4. Select the texture I've included above
5. Set interpolation to _closest_
6. Export the projects as glTF, limit to selected objects

# In Godot

1. Create a new 3D scene
2. Drag and drop the glb file created from Blender into the scene

### Minimal reproduction project

[closest-interpolation.zip](https://github.com/godotengine/godot/files/11037539/closest-interpolation.zip)

**Comment by fhd** [comment]
FWIW, while experimenting, I also noticed that changing extrapolation from _repeat_ (the default in Blender) to _extend_ has no effect in Godot. Inspecting the material on the actually imported object, the texture flags do have _repeat_ checked. If I uncheck it manually, it does not repeat, but the setting doesn't seem to be imported. Just mentioning it here since it seems related, but the issue that bothers me more is the _closest_ interpolation setting not working as described above.

**Comment by KoBeWi** [comment]
How it looks in Godot 4
![image](https://user-images.githubusercontent.com/2223172/227028336-4f09e6ca-378d-4e3c-bc1a-e37b5107eea6.png)
So I think it works?

**Comment by fhd** [comment]
Yes, that's what it should look like! Unfortunately I'm stuck with 3.x for now, for GLES2 support. But I suppose a fix _could_ be backported?
